examples/step-74/doc/results.dox

```diff
@@ -0,0 +1,195 @@
+<h1>Results</h1>
+
+The output of this program consist of the console output and
+solutions in vtu format.
+
+Convergence rate for the smooth case with polynomial degree 3:
+@code
+DEAL::Cycle 0
+DEAL::Number of active cells:       16
+DEAL::Number of degrees of freedom: 256
+   Solving system...
+DEAL::Writing solution to <sol_Q3-00.vtu>
+   Error in the L2 norm       :     0.00193285
+   Error in the H1 seminorm   :     0.106087
+   Error in the enery norm    :     0.150625
```

Comment:
This would be something useful to say in the introduction: That you compute errors against a known solution, evaluate convergence to the exact solution, etc.

Would it be worthwhile outputting the estimated error? It might be nice to compare it against the computed H1 seminorm error.